rust-toolchain

```diff
@@ -1 +1 @@
-1.77.2
```

Comment:
Memo: 
We need update rust toolchain as well as aws-config. https://github.com/smithy-lang/smithy-rs/pull/3940